package.json

```diff
@@ -13,58 +13,58 @@
   "author": "Evan Willhite <evan@fourkitchens.com>",
   "license": "MIT",
   "dependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
-    "@babel/polyfill": "^7.7.0",
-    "@babel/preset-env": "^7.6.3",
-    "@storybook/addon-a11y": "5.2.6",
-    "@storybook/addon-actions": "5.2.6",
-    "@storybook/addon-links": "5.2.6",
-    "@storybook/addons": "5.2.6",
-    "@storybook/react": "5.2.6",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.4",
+    "@babel/polyfill": "^7.8.3",
+    "@babel/preset-env": "^7.8.4",
+    "@storybook/addon-a11y": "5.3.13",
+    "@storybook/addon-actions": "5.3.13",
+    "@storybook/addon-links": "5.3.13",
+    "@storybook/addons": "5.3.13",
+    "@storybook/react": "5.3.13",
```

Comment:
Why are we pinning some of these requirements to specific versions instead of allowing non-breaking updates through semantic versioning? e.g. why isn't this `"^5.3.13"`?

When I run `yarn add @storybook/react` it puts the carat in to allow minor version updates. (it does the same with others, like @storybookaddons)

Reply:
I'm really not sure, but I think you're right. I'm going to add it.

Reply:
Done 👍